optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-backend/src/main/java/org/optaplanner/workbench/screens/solver/backend/server/SolverValidator.java

```diff
@@ -186,7 +188,7 @@ private String resolveProjectName(final Path resourcePath) {
 
     private String getSolverConfigResource(final Path resourcePath,
                                            final KieModule kieWorkbenchModule) {
-        return resourcePath.toURI().substring(kieWorkbenchModule.getRootPath().toURI().length() + "/src/main/resources/".length());
+        return Paths.convert(Paths.convert(resourcePath)).toURI().substring(Paths.convert(Paths.convert(kieWorkbenchModule.getRootPath())).toURI().length() + "/src/main/resources/".length());
```

Comment:
@mbarkley I'm just curious why do we need to convert both paths twice?

Reply:
Someone had removed the convert calls (which look like they should result in getting back the original `resourcePath`) but this broke the build, so I undid the change. I'm not sure why, but probably this merits a comment so that it is not accidentally changed again until the problem is investigated.

@ederign If you have time, could you add a comment for this?